src/components/Results/DetectedIssueResources.tsx

```diff
@@ -36,63 +51,66 @@ const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
               </Grid>
             </AccordionSummary>
             <AccordionDetails>
-              <Grid container item xs direction="row">
+              <Grid container item xs direction="row" spacing={2}>
                 <Grid container item xs={6} direction="column">
                   <Grid item xs>
-                    <h4>dataRequirement:</h4>
+                    <h4>Requirements:</h4>
                   </Grid>
-                  <Grid item xs>
-                    type: {fhirpath.evaluate(response, 'dataRequirement.type')}
+                  <Grid item xs direction="row">
+                    <Grid item xs>
+                      Type:{' '}
+                      <Link href={link.concat(fhirpath.evaluate(response, 'dataRequirement.type'))}>
+                        {fhirpath.evaluate(response, 'dataRequirement.type')}
+                      </Link>
+                    </Grid>
                   </Grid>
                   <Grid item xs>
-                    <h5>codeFilter:</h5>
+                    <h4>Codes:</h4>
+                    <h5>
+                      {codeFilterPath1[0].charAt(0).toUpperCase()}
+                      {codeFilterPath1[0].slice(1)}:
+                    </h5>
                   </Grid>
-                  <Grid>path: {fhirpath.evaluate(response, 'dataRequirement.codeFilter.path[0]')}</Grid>
                   <Grid item xs>
                     <Typography style={{ overflowWrap: 'break-word' }}>
-                      valueSet: {fhirpath.evaluate(response, 'dataRequirement.codeFilter.valueSet')}
+                      <Link href={fhirpath.evaluate(response, 'dataRequirement.codeFilter.valueSet')}>
+                        {fhirpath.evaluate(response, 'dataRequirement.codeFilter.valueSet')}
+                      </Link>
                     </Typography>
                   </Grid>
                   <Grid item xs>
-                    path: {fhirpath.evaluate(response, 'dataRequirement.codeFilter.path[1]')}
+                    <h5>
+                      {codeFilterPath2[0].charAt(0).toUpperCase()}
+                      {codeFilterPath2[0].slice(1)}:
+                    </h5>
```

Comment:
No need to `upperCase` here. We can keep the attribute name the same as is returned from the fhirpath